### ISSUE TITLE
Poor VoiceMailPackagingFunction defaults

### ISSUE THREAD

**Issue by vgw-chriskruger** [service-cloud-voice/ServiceCloudVoiceLambdas#43]
There seems to be a problem with the default configuration of the `VoiceMailPackagingFunction` lambda. The default lambda timeout is 60s as can be seen here in the yaml 

https://github.com/service-cloud-voice/ServiceCloudVoiceLambdas/blob/169a7180e329afeda947e4f61e1102bbe7f912b0/ServerlessApplication/ServiceCloudVoiceLambdas.yaml#L1440-L1454

However the function appears likely to always take longer than this because there is a default delay configured here (of 60s) and `delay_before_routing_vm_sec` isn't set or even mentioned in documentation.

https://github.com/service-cloud-voice/ServiceCloudVoiceLambdas/blob/169a7180e329afeda947e4f61e1102bbe7f912b0/voiceMailPackaging/handler.js#L13

It used down deeper in the handler here 

https://github.com/service-cloud-voice/ServiceCloudVoiceLambdas/blob/169a7180e329afeda947e4f61e1102bbe7f912b0/voiceMailPackaging/handler.js#L161-L178

Which, if I understand correctly is always going to try to wait blowing the default lambda timeout budget.

I suggest the following improvements:

1. A sensible default timeout if we want the function to wait _at least 60s_ in the handler. ~ 5 minutes based on MAX_ROUTING_ATTEMPTS = 4 and 60s default delay per try plus some time.
2. Update the documentation to discuss these delays and timeouts. 

The current default configuration is very likely to trip up everyone who uses it.

**Comment by jinalkathiara** [comment]
@vgw-chriskruger I agree with your assessment. We are planning to fix this issue by the Contact Center 13.
